guides/common/modules/proc_exporting-the-library-environment-syncable.adoc

```diff
@@ -0,0 +1,61 @@
+[id="Exporting_the_Library_Environment_in_a_Syncable_Format_{context}"]
+= Exporting the Library Environment in a Syncable Format
+
+You can export contents of all Yum repositories in the Library environment of an organization to a syncabe format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
+
+ifdef::satellite[]
+You can then serve the generated content via a local webserver on the importing  {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports. Instead on the importing {ProjectServer} you would need to
```

Comment:
```suggestion
You cannot directly import Syncable Format exports. Instead, on the importing {ProjectServer}, perform the following steps:
```

Shouldn't this be a "procedure" or something?  (Docs experts will know better..)